### PR TITLE
Adds margin, padding

### PR DESCRIPTION
**Pull request by vwallen**
Adds features from `css-box-4`:

- margin
- padding

### CODE REVIEW

**Review by jamesnw — APPROVED** [review]
I confirmed no `caniuse` features, and no `auto` keywords in BCD for padding. 🚀

File: features/margin.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: margin
+description: The `margin` CSS property sets space surrounding an element, and is a shorthand for `margin-top`, `margin-right`, `margin-bottom`, and `margin-left`.
```
Comment (captainbrosset) [inline features/margin.yml:2]:
```suggestion
description: The `margin` CSS property sets space around an element. It is a shorthand for `margin-top`, `margin-right`, `margin-bottom`, and `margin-left`.
```

File: features/padding.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: padding
+description: The `padding` CSS property sets space between an element's edge and its contents. It provides a shorthand for `padding-top`, `padding-right`, `padding-bottom`, and `padding-left`.
```
Comment (captainbrosset) [inline features/padding.yml:2]:
```suggestion
description: The `padding` CSS property sets space between an element's edge and its contents. It is a shorthand for `padding-top`, `padding-right`, `padding-bottom`, and `padding-left`.
```

**Review by captainbrosset — APPROVED** [review]
Some minor comments on descriptions, looks good to me otherwise.

**Review by ddbeck — APPROVED** [review]

**Pull request merged**